### DISCUSSION TITLE
How to change IntegrationParameters in bevy rapier2d

### DISCUSSION THREAD

**Discussion by derrick56007** [dimforge/rapier#438] [Q&A]
Hello, 

I am looking to be able to view a simulation at different speeds.. I have added

```rust
    .insert_resource(ScheduleRunnerSettings::run_loop(Duration::ZERO))
    .add_plugin(ScheduleRunnerPlugin {})
```
but I found this doesn't speed up the physics simulation, only the rendering speed.

How can I adjust the rapier2d setting such that the physics integration steps align with bevy loop rate? Thanks

**Comment by dror-g** [comment]
For future reference the `IntegrationParameters` can be set through `RapierContext` Resource:

```
fn set_physics_params(mut rapier_context: ResMut<RapierContext>) {
    rapier_context.integration_parameters.max_velocity_iterations = 2;
    rapier_context.integration_parameters.max_stabilization_iterations = 1;
    rapier_context.integration_parameters.max_velocity_friction_iterations = 2;
}
```
